assignments/simple-draw/code.js

```diff
@@ -1,46 +1,35 @@
-/*
- * This code is running in an environment with five functions defined:
- *
- *  drawLine(x1, y1, x2, y2, color, lineWidth)
- *
- *    Draws a line from x1,y1 to x2,y2 using the give color. The fifth argument,
- *    lineWidth, is optional and defaults to 1.
- *
- *  drawCircle(x, y, r, color, lineWidth=1)
- *
- *    Draws a circle centered at x,y with radius r using the given color. The
- *    fith argument, lineWidth, is optional and defaults to 1.
- *
- *  drawRect(x, y, width, height, color, lineWidth=1)
- *
- *    Draws a rectangle starting at x,y with the given width, height, and color.
- *    Positive widths go to the right and negative to the left; positive heights
- *    go down and negative heights go up. The sixth argument, lineWidth, is
- *    optional and defaults to 1.
- *
- *  drawFilledRect(x, y, width, height, color)
- *
- *    Draws a filled rectangle starting at x,y with the given width, height, and
- *    color. Positive widths go to the right and negative to the left; positive
- *    heights go down and negative heights go up.
- *
- *  drawFilledCircle(x, y, r, color)
- *
- *    Draws a filled circle centered at x,y with radius r using the given color.
- *
- * There are also a couple variables that might be useful:
- *
- *  width - the width of the drawing area.
- *  height - the height of the drawing area.
- *
- */
+drawFilledRect (0, 0, 500,600, 'rgba(115,215,255,1)')
+drawFilledRect (0,(2*height/3)+(height/6), width, (2*height/3), 'rgba(0, 64, 0, 1')
+let rect = 1000
+while (rect>= 2*height/3) {
+  drawLine (0, rect, width, rect, 'rgba(0,128,0, 1', 0)
+  rect -= 2
+  while (rect >= 3*height/4){
+  drawLine (0, rect, width, rect, 'rgba(0,128,0, 1', 0)
+  rect --
```

Comment:
One last style point: there should be no space between the variable name and an operator that only takes one operand (a so-called "unary" operator) like `--`. So:

```suggestion
  rect--
```